### DISCUSSION TITLE
Modify RPI-RP2 drive

### DISCUSSION THREAD

**Discussion by LazaroFilm** [earlephilhower/arduino-pico#1913] [Q&A] [answered]
Is it possible to modify the `RPI-RP2` drive that loads up when in boot loader mode? 
- I would love to be able to change the `INDEX.HTM` to my project's page. 
- Ideally, I would also like to modify the name of the drive to my project's name.

**Comment by earlephilhower** [comment, marked as answer]
Nope, that's actually burned into the onboard ROM.  See the [repo here](https://github.com/raspberrypi/pico-bootrom).

**Reply by LazaroFilm** [reply]
Ah shucks. Thanks for the prompt answer and Happy Holidays!
